product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/09_11610_rel_notes.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: "Version 1.16.1.0"
+---
+
+New features, enhancements, bug fixes, and other changes in Pgbouncer 1.16.1.0 include the following:
+
+| Type         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| ------------ |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Security Fix | Make PgBouncer acting as a server reject extraneous data after an SSL or GSS encryption handshake.<br />A man-in-the-middle with the ability to inject data into the TCP connection could stuff some cleartext data into the start of a supposedly encryption-protected database session. This could be abused to send faked SQL commands to the server, although that would only work if PgBouncer did not demand any authentication data. However, a PgBouncer setup relying on SSL certificate authentication might well not do so.|
```

Comment:
Typically we want to use past tense for describing bug fixes. 
Is "stuff" the appropriate technical term?
Add commas around the phrase "acting as a server"

Reply:
Remove the word "well".in the last sentence.